.github/workflows/tests.yaml

```diff
@@ -115,3 +117,6 @@ jobs:
           pytest -v --color=yes --doctest-only geopandas --ignore=geopandas/datasets
 
       - uses: codecov/codecov-action@v4
+
+    # skip scheduled runs on forks
+    if: (github.event_name == 'schedule' && github.repository == 'geopandas/geopandas') || (github.event_name != 'schedule')
```

Comment:
Can we move it closer to the top of the job definition? It is super hidden here.